### PR TITLE
Fixed tests affected by tabletools in the built version

### PR DESCRIPTION
**Pull request by mlewand**
Some simply were failing because of ACF transformations, the other was failing because tabletools was adding an extra CSS file. Also added extra safety check, which was failing in one of the tests.

Fixes tests:
* http://tests.ckeditor.dev:10470/tests/plugins/wysiwygarea/contentscss
* http://tests.ckeditor.dev:10470/tests/plugins/tableselection/integrations/clipboard/pastenested
* http://tests.ckeditor.dev:1030/tests/plugins/tableselection/integrations/core/style

### CODE REVIEW

File: plugins/tableselection/plugin.js [hunk 1]
```diff
@@ -761,7 +761,7 @@
 							targetNode = targetNode.getPreviousSourceNode( false, CKEDITOR.NODE_ELEMENT, boundaryTable );
 						}
 
-						if ( !targetNode && !endNode.is( 'table' ) && endNode.getNext() ) {
+						if ( !targetNode && endNode && endNode.is && !endNode.is( 'table' ) && endNode.getNext() ) {
```
Comment (Comandeer) [inline plugins/tableselection/plugin.js]: I don't understand why this change is added in this PR.

Comment (mlewand) [inline plugins/tableselection/plugin.js]: When I ran all the tests, this fragment was failing for one TC (don't remember which one exactly), the thing was that endNode was a textNode - so it did not have `is` method. I decided to add it as a just extra safety.

Comment (Comandeer) [inline plugins/tableselection/plugin.js]: Ok, fair enough ;)
